src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -15,13 +15,16 @@ dotnet add package OpenTelemetry.Exporter.Zipkin
 
 ## Configuration
 
+You can enable the the `ZipkinExporter` with the `AddZipkinExporter()`
+[extension method](https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterHelperExtensions.cs#L35).
```

Comment:
I'm worried about the link here since it could be affected by the code change.
A permanent link won't have such problem but it also creates other problems.

Reply:
I think we can remove this link, and simply state that ...with the `AddZipkinExporter()`.. Further down this doc is link to example code, which should help.

Reply:
removed link per suggestion